### PR TITLE
Update to run on x86

### PR DESCRIPTION
**Pull request by elvece**

### CODE REVIEW

**Comment by elvece** [comment]
Proof:
```
root@embassy-<redacted>:/home/start9# docker exec -it gitea.embassy /bin/bash
bash-5.1# uname -a
Linux gitea.embassy 5.10.92-v8+ #1514 SMP PREEMPT Mon Jan 17 17:39:38 GMT 2022 x86_64 Linux
```

File: README.md [hunk 1]
```diff
@@ -104,5 +102,11 @@ you can install simply by running:
...
 ```
 
-**Tip:** You can also install the gitea.s9pk using **Sideload Service** under
-the **Embassy>SETTINGS** section.
+> **Tip:** You can also install the gitea.s9pk using **Sideload Service** under
+the **Embassy > Settings** section.
+
+## Verify Install
+
+Go to your Embassy Services page, select **Gitea**, configure and start the service. Then, verify it's interfaces are accessible.
```
Comment (chrisguida) [inline README.md:110]:
```suggestion
Go to your Embassy Services page, select **Gitea**, configure and start the service. Then, verify its interfaces are accessible.
```

File: README.md [hunk 2]
```diff
@@ -104,5 +102,11 @@ you can install simply by running:
 make install
 ```
 
-**Tip:** You can also install the gitea.s9pk using **Sideload Service** under
-the **Embassy>SETTINGS** section.
+> **Tip:** You can also install the gitea.s9pk using **Sideload Service** under
+the **Embassy > Settings** section.
+
+## Verify Install
```
[the remark this answers is not in the document]
Comment (chrisguida) [reply README.md:108]: Also there's already a section called "Installing", perhaps just omit this header

[the remark this answers is not in the document]
Comment (elvece) [reply README.md:108]: Will make it a sub section of "Installing"

File: Makefile [hunk 3]
```diff
@@ -1,28 +1,36 @@
...
 clean:
+	rm -rf docker-images
+	rm -f  $(PKG_ID).s9pk
 	rm -f image.tar
-	rm -f gitea.s9pk
+	rm -f scripts/*.js
 
-gitea.s9pk: manifest.yaml image.tar instructions.md scripts/embassy.js
+ $(PKG_ID).s9pk: manifest.yaml instructions.md scripts/embassy.js gitea/LICENSE docker-images/aarch64.tar docker-images/x86_64.tar
+	if ! [ -z "$(ARCH)" ]; then cp docker-images/$(ARCH).tar image.tar; fi
```
Comment (chrisguida) [inline Makefile:24]: the two lines above this have extra whitespace at the beginning

File: Makefile [hunk 4]
```diff
@@ -1,28 +1,36 @@
...
 
 clean:
+	rm -rf docker-images
+	rm -f  $(PKG_ID).s9pk
 	rm -f image.tar
-	rm -f gitea.s9pk
+	rm -f scripts/*.js
 
-gitea.s9pk: manifest.yaml image.tar instructions.md scripts/embassy.js
+ $(PKG_ID).s9pk: manifest.yaml instructions.md scripts/embassy.js gitea/LICENSE docker-images/aarch64.tar docker-images/x86_64.tar
```
Comment (chrisguida) [inline Makefile:23]: There's a space at the beginning of this line, which causes Make to puke

Comment (elvece) [reply Makefile:23]: fixed

**Comment by chrisguida** [comment]
Looks good

**Pull request merged**